packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,165 @@
+Okta which handles the flow in it's own window, or handle each step in the flow by
+building the appropriate user interface elements, capturing the information, and updating
+Okta.
+
+This guide is an overview of building your own user interface. For information on the redirect model, see <StackSnippet snippet="redirectquickstart" inline />.
+
+## Sign-in flow
+
+Okta supports many ways of authenticating the identity of a user during sign-in. An Okta org administrator creates _policies_, different mixes of authentication methods (also
+called factors), and assigns them to apps, people, groups, and more. Policies also
+configure how many authentication methods are required (multifactor), and which methods
+are optional. This results in a large number of possible combinations for the different
+factors and steps to complete the sign-in flow.
+
+The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine.
+You initialize the machine with the details of your Okta Application Integration, request
+the initial step in the flow, and cycle through responding to steps until either the user
+is signed in, cancels, or an error occurs.
+
+<div class="common-image-format">
+
+![A diagram showing the sign-in flow.](/img/mobile-sdk/mobile-idx-basic-flow.png "A diagram that shows the sign-in flow.")
+
+</div>
+
+Each sign-in step may include one or more possible user actions, such as:
+- Choosing an authenticator, such as Okta Verify or WebAuthN.
+- Entering a One Time Passcode (OTP).
+- Cancelling the sign-in.
+
+## Sign-in objects
+
+Each step in the flow is represented by a _Response_ which contains the information used
+for creating the UI and general state information. The response is also used to cancel the
+sign-in, or retrieve an access token after the sign-in succeeds.
+
+The UI information is divided between two objects in the response, authenticators and
+remediations. An _Authenticator_ represents a type, or factor for verifying the identity
+of a user, such as a username and password, or Okta Verify. A _Method_ represents a
+channel for an authenticator, such as using email or SMS for an OTP. An authenticator may
+have multiple methods. A _Remediation_ represents a user action and usually contains most
+of the information for a step. Each response may contain multiple remediations and
+authenticators.
+
+Some common types of user action are represented by a _Capability_. These include
+requesting a new OTP, or a password reset request. Remediations, authenticators, and
+methods may contain capabilities.
+
+A _Form_ inside the remediation represents the user action in a collection of _Field_
+objects. Most fields represent something that's displayed in the UI, either a static
+element, such as a label, or a user input, such as a selection list. The field also
+contains state information, such as whether the associated value is required or secret.
```

Comment:
Thanks, I'm going to remove "or secret" because it would take longer to describe properly. It's usually associated with a text entry field for passwords and similar and means that the contents of the field are "secure." (and I didn't like that explanation either!) Fix is in the next commit.